### PR TITLE
add unit-test for ValidateCPUPeriod

### LINKED ISSUE #1637: [help wanted] add unit-test for ValidateCPUPeriod
### Ⅰ. Issue Description
Add unit-test for ValidateCPUPeriod method which locate on `apis/opts/cpu.go`.

You can take [env_test.go](https://github.com/alibaba/pouch/blob/master/apis/opts/env_test.go) for reference.

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1.
2.
3.

### Ⅴ. Anything else we need to know?
please "Baiji" developer refer to this issue in your PR for the convenience for us to count your score.

### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by youmengna**
### Ⅰ. Describe what this PR did
add unit-test for ValidateCPUPeriod

### Ⅱ. Does this pull request fix one issue?

fixes #1637

### CODE REVIEW

File: apis/opts/cpu_test.go [hunk 1]
```diff
@@ -5,6 +5,26 @@ import (
...
+	tests := []struct {
+		name    string
+		period  int64
+		wantErr bool
+	}{
+		// TODO: Add test cases.
+		{name: "test1", period: 0, wantErr: false},
+		{name: "test2", period: 10, wantErr: true},
+		{name: "test3", period: 2000000, wantErr: true},
+		{name: "test4", period: 1000, wantErr: false},
```
Comment (allencloud) [inline apis/opts/cpu_test.go]: How about adding some boundary value here in the test case, like 1000 and 100000? @youmengna

**Comment by allencloud** [comment]
LGTM
